cyences_app_for_splunk/default/macros.conf

```diff
@@ -489,6 +489,10 @@ iseval = 0
 definition = search *
 iseval = 0
 
+[cs_azure_securityscore]
+definition = index=*
```

Comment:
Bro, lets use o365 index. That way we'll avoid index=*. Also, please Cyences doc change that I did to make sure I added correct information about this.

Reply:
bro we used SourceType in the search query so it will not use much resource. But I will change the index.